sycl/include/CL/sycl/buffer.hpp

```diff
@@ -370,6 +370,10 @@ class buffer {
       return newRange[0] == 1 && newRange[2] == parentRange[2];
     return newRange[1] == parentRange[1] && newRange[2] == parentRange[2];
   }
+
+  static_assert (dimensions > 0 && dimensions <= 3,
+		  "Range of the buffer can only be 1/2/3 dimensional.");
```

Comment:
```suggestion
		  "A SYCL buffer can only be 1/2/3 dimensional.");
```
What does "Range of the buffer" mean?